src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -66,5 +66,8 @@ public CommandResult execute() {
             return new CommandResult(MESSAGE_DUPLICATE_PERSON);
         }
     }
+    public boolean isMutating() {
```

Comment:
You should have a @Override here, which would be safer and increase readability.